manager/src/main/java/com/redhat/service/smartevents/manager/api/models/responses/ProcessorResponse.java

```diff
@@ -36,7 +37,10 @@ public ProcessorResponse() {
     private String transformationTemplate;
 
     @JsonProperty("action")
-    private BaseAction action;
+    private Action action;
```

Comment:
@kostola I think this violates my expectations as a user of the API. For example, I submit this:

```
 {
    "name": "slack-source",
    "source": {
        "type": "Slack",
        "parameters": {
            "channel": "channel-to-read-from",
            "token": "xoxb-..."
        }
    }
}
```
but I then get back something with an `Action` in it:

```
{
    "name": "slack-source",
    "source": {
        "type": "Slack",
        "parameters": {
            "channel": "channel-to-read-from",
            "token": "xoxb-..."
        }
    },
   "action": "{
       "type" : "sendToBridge",
       ...
    }
}
```

The `action` is a purely internal implementation detail of how we have decided to currently ingest events into the bridge instance. The user shouldn't be aware of it IMHO.

My expectation as a user is that I get back something that represents only what I submitted e.g a `Source` configuration.

Reply:
Ignore me. It doesn't return the resolved action.

Reply:
No, it does :-)

IMHO it shouldn't because it' a purely implementation detail.